api/swagger-spec/v1.json

```diff
@@ -21674,6 +21674,51 @@
      }
     }
    },
+   "v1.RBDVolumeSource": {
```

Comment:
why do we have both RBDPersistentVolumeSource and this ?

Reply:
Because only the persistent volume version lets you reference a secret in a namespace other than the pod.